docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,100 @@
+Certora Prover Conf File API
+============================
+
+The **Conf File API** is an alternative API for  `certoraRun` . In terms of functionality 
+this API is identical to the [CLI Options](options.md) API. Instead of calling `certoraRun` 
+with a list of shell flags, some or all the flags can be stored in a [JSON](https://www.json.org/json-en.html) file 
+(to be more precise the format is [JSON5](https://json5.org/)):
+
+```bash
+certoraRun my_params.conf
+```
+<div style="background-color: #FFFFE0; padding: 10px; border: 1px solid #E6DB55;">
+To denote a text file as a conf file use the <strong>.conf</strong> suffix
+</div>
+
+Converting from CLI Options to JSON
```

Comment:
This section should say how to automatically get the conf from a runscript.

Reply:
I renamed the title to "How CLI Options are mapped to JSON". I later described where to find the generated conf file